uvicorn/main.py

```diff
@@ -225,6 +242,9 @@
     multiple=True,
     help="Specify custom default HTTP response headers as a Name:Value pair",
 )
+@click.option(
+    "--version", is_flag=True, callback=print_version, expose_value=False, is_eager=True
```

Comment:
Looks great, yup! I think we just want to add something like `help="Display the uvicorn version and exit."` here.